src/main/java/uk/gov/hmcts/darts/util/DataUtil.java

```diff
@@ -9,7 +9,7 @@
 
 public final class DataUtil {
     private DataUtil() {
-
+        // Utility class
     }
```

Comment:
Consider removing the private constructor, and instead annotate the class with Lombok's `@UtilityClass` which will generate it for us. That is the pattern generally used elsewhere for util classes.

Reply:
done